### PR TITLE
Deputies (Departmental mall-cops)

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Adds in the deputy role.
Deputies are now a midpoint between crew and security officers, acting as a departmental mall-cop.
Deputies are mindshielded and are protected from gang, cult, revolution and hivemind. They are not protected from other antagonist roles.
While regular security officers have better training and make proper arrests acting like a station security force, deputies act as a mall cop, preventing minor crimes in their area and handing off criminals to proper security forces.

Playtime as security will be changed from time as a crew member to 4 hours as detective / deputy and deputy will require 3 hours as regular crew and 7 days since joined. Deputies are essentially the trainees, that aren't quite proper security yet.

Regular security officers will never be assigned to a department now and will always be security that patrol the halls and respond to deputy emergency calls.

Deputy gear is very weak and it is recommended they call in for backup for any serious threats. They are there to monitor departments and ensure no suspicious activity / anti social behaviour rather than counter the cult hiding out in maints.

They spawn in their departmental outposts.

This will probably need some SOP changes.

A major point of them is to act as a stepping stone between sec and crew members and make it so there isn't so much of a harsh divide between full meta protection and the regular suspicious people. The line of trust gradually increases. Deputies are somewhat protected but are still vulnerable unlike security. Admittidly there would be no point to them if they were meta-protected as they would just be security again.

### Roles of the deputy
Deputies will always be assigned to a department and will have the job of preventing unauthorised entry to that department.
Deputies do not have the ability to make proper arrests as they do not have access to brig cells (excluding lowpop). Instead they will need to hand off captured criminals to regular security or issue citations.
They follow the orders of the department that they are stationed in, or the HOS.

### Gear of the deputies
Since they aren't there to make proper arrests, deputies do not get access to sec-huds.
Deputies have a unique police baton varient that deals 35 stamina damage and has 5 force instead of 12. The baton is pretty weak and very difficult to make an arrest on someone if they are running away, however useful for defensive sitations such as when you are in your department.
They have access to basic brig doors, but not the gear or lockers of normal security.

### Emergency!
On red alert deputies gain access to security lockers and brig cells.

## Why It's Good For The Game

A new role that has less responsibilities than a normal security officer and is a lot more chill.
A lot weaker than regular security and there to help out with annoying people around the departments rather than handle anything serious.
No more role on the job menu that does absolutely nothing.

## Changelog
:cl:
add: Adds in deputies: departmental mall-cops.
config: Config of security default spawn amoutn changed from 10 to 5. Scaling of security spawns changed from 12 max to 8 max.
config: Deputy spawn amount changed from 0 to 4.
/:cl:

### CODE REVIEW

File: config/jobs.txt [hunk 1]
```diff
@@ -41,9 +41,9 @@ Roboticist=2,2
 
 Warden=1,1
 Detective=1,1
-Security Officer=10,8
```
Comment (PowerfulBacon) [inline config/jobs.txt:44]: Not sure why this was at 10, someone must have got a little overzelous with security and not noticed the security scaling coefficient.

**Comment by Vexylius** [comment]
Please don't make the detective an entry job for security. If it is already, then it shouldn't be

**Comment by PowerfulBacon** [comment]
>  If it is already, then it shouldn't be

It already is

**Comment by Raven-Industries** [comment]
I have 2 concerns about this: First, adding another sec role might spread players out a bit thin; as it is now, on a round with 50-55 players, it's rare that you have a sec officer for each department. With a new role, you're going to have people spread out into both deputies and sec officers, instead of just officers, and with officers no longer having department specific access, you're going to get even less coverage than usual. 
For this, maybe instead of making deputy a separate role, it could be a optional subtype of other jobs? Ie, in character selection, you can choose medical doctor or paramedic, but then toggle "can be deputy" to on and then you play as a doctor with the additional task of department security. Maybe have them spawn with a red armband and the baton and provide basic security _in _addition__ to their regular job. 

The other potential concern (which holds even with the above proposal) is that sec officers will have nothing to do now. Without department access, they're limited to the brig (which isn't the sort of place that stays occupied long-term, there's not really a task to keep you there like with the other departments) or public areas and hallways, which is fine and all, but you can only patrol the hallways so much before it gets boring. You also lose some the inter-department interaction (and conflict) that comes with having a personal representative of Nanotrasen interests with full arrest powers permanently stationed in your department.

**Comment by PowerfulBacon** [comment]
Sec officers will absolutely not have nothing to do. Security officers are the ones that handle arrests on the station and take orders from the HOS for more serious events. They will act as regular security officers, the way they normally do.

As for spreading the sec force too thin, this shouldn't be an issue. During lowpop deputies gain the ability to access brig cells and the HOS / warden can always promote them to a higher level if needed. Deputies are specifically for players who just want to chill out and avoid playing security anyway due to the stress of constantly having to go after people. While regular security officer is there for those players who want to play how security act normally. The security force not having their advanced departmental accesses wont be a major issue since only 1 officer could get departmental access and in the majority of cases the security officer is too busy to be stationed in that department.

**Comment by ike709** [comment]
Haven't gone over the code but I approve of the concept.

**Comment by BrumalLake** [comment]
mindshield? kinda liked not getting gibbed by cultists or boned by revolutionaries though
was kinda hoping for a midpoint between sec and conversions like brig phys or lawyer (exactly like the old deputy)

**Comment by PowerfulBacon** [comment]
I don't mind if they are mindshielded or not. I guess it would make more sense for them not to be. They would still need to be rev-proof which is the only thing that could screw them up if the HOS mindshields them.

**Comment by BrumalLake** [comment]
> 
> They would still need to be rev-proof which is the only thing that could screw them up if the HOS mindshields them.

they really don't same case for lawyers and brig physes
all 3 get sunglasses anyways assuming they get the deputy hud
if this is about starting as a head rev being your meaning then ok i guess

**Comment by JohnFulpWillard** [comment]
Can’t wait for Bee players on Fulp to mhelp asking why they don’t have sec radio

**Comment by PowerfulBacon** [comment]
> 
> 
> Can’t wait for Bee players on Fulp to mhelp asking why they don’t have sec radio

F*lp

**Comment by Boxlta** [comment]
They should get a mindshield lite, which blocks rev, but not other antags. I think it would honestly be better if they could be cult or what not

**Comment by Sapfearon** [comment]
they should be antag proof if they are for new players who wants to be security. Again, i will always in favor of blocking any heads/sec from rolling antags. currently brig phys is most broken antag role you can get and all this due to easy access to brig, armory and being part of security team even if they are not supposed to be.

**Comment by Kapu1178** [comment]
Skyrat did something like this. Its terrible. Please no.

**Pull request closed without merging**